DBContextHelper/DBContextHelper.Tests.Integration/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+﻿Copyright (c) 2006 Damien Miller <djm@mindrot.org> (jBCrypt)

```

Comment:
Don't think this file should be here.  Should be something like a license folder and it go in there if we are really using this.